Extension point called very early before the main DITA OT processing

raducoravu opened the pull request:
Possible suggestion for: https://github.com/dita-ot/dita-ot/issues/3094

Signed-off-by: Radu Coravu <radu_coravu@oxygenxml.com>

robander commented:
I like this idea -- @jelovirt any concerns?

@raducoravu can you update the commits to add the sign-off?

raducoravu commented:
@robander Updating the commit messages seems to be quite difficult once they are pushed. If fix looks OK, maybe I can cancel this PR and open a new one on another branch.

raducoravu commented:
@infotexture Thanks but I have two commits that I need to modify and they are not even my last commits as I also updated the branch and that appears to be my last commit. Also I know modifying comment messages can be done for older commits, I read the tutorial for this on GitHub, tried to do that but failed. 
I give up, somehow I always seem to get various things wrong when contributing pull requests to the DITA OT. 
Anyway I'm closing the pull request, I added some fix suggestions to the opened issue as this is not something I urgently need.
